thirdeye/thirdeye-frontend/app/utils/anomaly.js

```diff
@@ -1,6 +1,68 @@
 import moment from 'moment';
 import { isPresent } from '@ember/utils';
 import _ from 'lodash';
+import {
+  checkStatus,
+  postProps
+} from 'thirdeye-frontend/utils/utils';
+import fetch from 'fetch';
+
+/**
+ * Response type options for anomalies
+ */
+export const anomalyResponseObj = [
+  { name: 'Not reviewed yet',
+    value: 'NO_FEEDBACK',
+    status: 'Not Resolved'
+  },
+  { name: 'True anomaly',
+    value: 'ANOMALY',
+    status: 'Confirmed Anomaly'
+  },
+  { name: 'False alarm',
+    value: 'NOT_ANOMALY',
+    status: 'False Alarm'
+  },
+  { name: 'Confirmed - New Trend',
+    value: 'ANOMALY_NEW_TREND',
+    status: 'New Trend'
+  }
+];
+
+/**
+ * Returns the anomaly data url
+ * @param {Number} startStamp - the anomaly start time
+ * @param {Number} endStamp - the anomaly end time
+ * @returns {String} the complete anomaly data url
+ * @example getAnomalyDataUrl(1491804013000, 1491890413000) // yields => /anomalies/search/anomalyIds/1491804013000/1491890413000/1?anomalyIds=
+ */
+export function getAnomalyDataUrl(startStamp = 0, endStamp = 0) {
+  return `/anomalies/search/anomalyIds/${startStamp}/${endStamp}/1?anomalyIds=`;
```

Comment:
👍 fyi, I believe Steve started an API file, we can start thinking about putting new urls there in an effort to  alleviate new tech debt

Reply:
He has it for self-serve only. I can put another one for maybe something general.

`app/utils/api/self-serve.js`

Reply:
added `utils/api/anomaly.js`. DONE